test-case/check-pause-release-suspend-resume.sh

```diff
@@ -74,7 +76,7 @@ OPT_HAS_ARG['i']=1             OPT_VAL['i']='500'
 OPT_NAME['s']='sof-logger'   OPT_DESC['s']="Open sof-logger trace the data will store at $LOG_ROOT"
 OPT_HAS_ARG['s']=0             OPT_VAL['s']=1
 
-OPT_NAME['t']='tplg'     OPT_DESC['t']='tplg file, default value is env TPLG: $TPLG'
+OPT_NAME['t']='tplg'     OPT_DESC['t']='tplg path, default is environment variable: TPLG'
```

Comment:
TPLG is the path+filename, not just the path. This doesn't seem like a valid correction.

Reply:
Moreover this is a correction of a previous commit in this same PR. The git history is still a mess, don't waste your time reviewing each commit. In addition to having commits that can't run, history that does not bisect, reverts or cherry-picks and confused reviewers, this is also confusing github: so you can't answer some threads unless you select the right commit.

Back to the description: "path" is ambiguous, for some people path the `dirname` but for many others like https://docs.python.org/3/library/pathlib.html path = dirname + filename.

"file" is vague but at least every knows it's vague, so no one makes wrong assumptions about it. So I agree "file" is better.

TPLG supports both complete paths and filenames.